src/WindowManager.cc

```diff
@@ -504,6 +504,10 @@ void
 WindowManager::startBackground(const std::string& theme_dir,
                                const std::string& texture)
 {
+    if (!pekwm::config()->getThemeBackground()) {
```

Comment:
I think the stopBackground should be left intact and then just do

```
stopBackground();
if (pekwm::config()->getThemeBackground() && ! texture.empty()) {
```

Reply:
Ah yes, for reload case (from theme background to none) I guess. Fixed.